docs/HowTo/UpdatePatroni.md

```diff
@@ -0,0 +1,123 @@
+# Updating Patroni from 1.6.5 to 2.0.1
+
+**Please note: while this fix has been testing multiple times in a lab environment, the nature of the bug means that I cannot guarantee that this fix will be completely outage-free. I did repeat the test multiple times and was unable to cause an outage using this method, but this may depend somewhat on your particular deployment. Therefore, please ensure that you test sufficiently in your test environment before applying the upgrade to prod!**
```

Comment:
```suggestion
**Please note: while this fix has been testing multiple times in a lab environment by the Platform Services Team the nature of the bug means that we cannot guarantee that this fix will be completely outage-free. We did repeat the test multiple times and was unable to cause an outage using this method, but this may depend somewhat on your particular deployment. Therefore, please ensure that you test sufficiently in your test environment before applying the upgrade to prod!**
```